Subscriber/CheckoutSubscriber.php

```diff
@@ -381,7 +381,7 @@ private function addAdyenGooglePay(Shopware_Controllers_Frontend_Checkout $subje
                 'merchantName' => Shopware()->Shop()->getName()
             ],
         ];
-        if ($this->configuration->getEnvironment() === Configuration::ENV_LIVE) {
+        if ($this->configuration->getEnvironment() === strtolower(Configuration::ENV_LIVE)) {
             $adyenGoogleConfig['environment'] = 'PRODUCTION';
```

Comment:
Hi @isfett thanks for spotting this and thanks for the PR.
I would recommend changing the [`Configuration::ENV_LIVE`](https://github.com/Adyen/adyen-shopware5/blob/cb03b94a3c9a57d705e2239c6a508369defe2843/Components/Configuration.php#L21) constants instead, so that they are aligned with the constants in our PHP library.
What do you think @acampos1916 ?

Reply:
Would be fine, but there have to be some more changes than: The ApiFactory has the same error and can be fixed with this change, the $lowercase parameter will be not needed any more and old calls including $lowercase = true needs to be changed. would be better if you could do those changes if you want to change the constants to lowercase instead.

Reply:
Thanks for you input @isfett , you're right. I'll change the constants and the affected code.

Reply:
@peterojo I agree on switching these to lowercase. Thanks @isfett!